pkgs/tools/text/html-tidy/default.nix

```diff
@@ -11,9 +11,18 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-ZMz0NySxzX2XHiqB8f5asvwjIG6kdIcq8Gb3EbAxBaU=";
   };
 
-  nativeBuildInputs = [ cmake libxslt/*manpage*/ ];
+  # https://github.com/htacg/tidy-html5/pull/1036
+  patches = (fetchpatch {
+    url = "https://github.com/htacg/tidy-html5/commit/e9aa038bd06bd8197a0dc049380bc2945ff55b29.diff";
+    sha256 = "sha256-Q2GjinNBWLL+HXUtslzDJ7CJSTflckbjweiSMCnIVwg=";
+  });
 
-  cmakeFlags = [];
+  nativeBuildInputs = [ cmake libxslt/*manpage*/ ]
```

Comment:
We are missing manpage here?

Reply:
No, that comment just says that libxslt required for building man page.